modules/bootstrap-ks/Makefile

```diff
@@ -0,0 +1,353 @@
+#-- bootstrap-ks Params---#
+BOOTSTRAP_KS_REPO ?= https://${GITHUB_TOKEN}@github.com/open-cluster-management/bootstrap-ks.git
+BOOTSTRAP_KS_BRANCH ?= master
+BOOTSTRAP_KS_DEPLOY_DIR ?= bootstrap-ks
+
+#---Login/Host-cluster related env vars---#
+# API URL of the cluster running the target clusterpool, used in oc login, so port :6443 is required
+BOOTSTRAP_KS_HOST_API ?=
+# Access credentials used to log in to cluster indciated in BOOTSTRAP_KS_HOST_API
+BOOTSTRAP_KS_HOST_USER ?=
+BOOTSTRAP_KS_HOST_PASS ?=
+BOOTSTRAP_KS_HOST_TOKEN ?=
+# Namespace where Hive ClusterPool resources reside
+BOOTSTRAP_KS_HOST_NAMESPACE ?= clusterpool
+
+OC_CLUSTER_USER = BOOTSTRAP_KS_HOST_USER
+OC_CLUSTER_PASS = BOOTSTRAP_KS_HOST_PASS
+OC_CLUSTER_API = BOOTSTRAP_KS_HOST_API
+
+CLUSTER_NAME_PREFIX ?= canary
```

Comment:
I don't see this referenced anywhere else?

Reply:
Both `OC_CLUSTER_PASS = BOOTSTRAP_KS_HOST_PASS` and `OC_CLUSTER_API = BOOTSTRAP_KS_HOST_API` are used by the OC (https://github.com/open-cluster-management/build-harness-extensions/blob/master/modules/oc/Makefile) module.

`CLUSTER_NAME_PREFIX` is used in the *KS provision.sh stuff when building the cluster name